src/main/java/mate/jdbc/model/Manufacturer.java

```diff
@@ -1,70 +1,19 @@
 package mate.jdbc.model;
 
-import java.util.Objects;
+import lombok.Data;
+import lombok.NoArgsConstructor;
 
+@Data
+@NoArgsConstructor
 public class Manufacturer {
     private Long id;
     private String name;
     private String country;
-
-    public Manufacturer() {
-    }
+    private boolean isDeleted;
 
     public Manufacturer(Long id, String name, String country) {
```

Comment:
Since you are using Lombok's @NoArgsConstructor, consider removing the constructor with parameters and use Lombok's @AllArgsConstructor instead.